ssl/record/rec_layer_d1.c

```diff
@@ -559,16 +559,13 @@ int dtls1_read_bytes(SSL *s, uint8_t type, uint8_t *recvd_type,
 
     if (!ossl_statem_get_in_handshake(sc)
         && (rr->type == SSL3_RT_HANDSHAKE || rr->type == SSL3_RT_ACK)) {
+        int ined = (sc->early_data_state == SSL_EARLY_DATA_READING);
```

Comment:
These changes to this function dtls1_read_bytes mimics what was there for TLS1.3. WIthout them handshake packets would not be processed when we receive them after early data.